system_metrics_collector/test/system_metrics_collector/test_periodic_measurement_node.cpp

```diff
@@ -55,6 +59,16 @@ class TestPeriodicMeasurementNode : public ::system_metrics_collector::PeriodicM
     return times_published_;
   }
 
+  /**
+   * Return true if the lifecycle publisher is activated, false if null or not activated.
+   *
+   * @return
+   */
+  bool IsPublisherActivated() const
+  {
+    return !(publisher_ == nullptr || !publisher_->is_activated());
```

Comment:
There is a lot of negation going on here, making the logic unclear. How about:
```suggestion
    return publisher_ != nullptr && publisher_->is_activated();
```

Reply:
Sure